_test_common/lib/test_phases.dart

```diff
@@ -70,12 +68,11 @@ void _printOnFailure(LogRecord record) {
 Future<BuildResult> testBuilders(
   List<BuilderApplication> builders,
   Map<String, /*String|List<int>*/ Object> inputs, {
+  BuildResult? resumeFrom,
```

Comment:
Why do it this way? The whole Expando thing seems like somewhat of a hack, and unless there's a good reason I'd prefer not to use it.

Reply:
Thanks, done.

I was trying to avoid changing the return type to avoid such a big breaking change to tests ... but I was confused, I thought this was in `build_test` which is published. But fortunately it's in `_test_common` which we can just change. So, changed the return type :)